input_parser.star

```diff
@@ -27,8 +27,8 @@ DEFAULT_DEPLOYMENT_STAGES = {
 
 DEFAULT_IMAGES = {
     "agglayer_image": "ghcr.io/agglayer/agglayer-rs:pr-96",  # https://github.com/agglayer/agglayer/pkgs/container/agglayer-rs
-    "cdk_erigon_node_image": "hermeznetwork/cdk-erigon:ab3013d",  # https://hub.docker.com/r/hermeznetwork/cdk-erigon/tags
-    "cdk_node_image": "ghcr.io/0xpolygon/cdk:0.3.0-beta4",  # https://github.com/0xpolygon/cdk/pkgs/container/cdk
+    "cdk_erigon_node_image": "jerrycgh/cdk-erigon:fast_witness_generation_v4",  # https://hub.docker.com/r/hermeznetwork/cdk-erigon/tags
```

Comment:
Not 100% sure if the new image is the preferred version, but maybe we could stick with equivalent version from `hermeznetwork` for consistency

Reply:
I would prefer that. At the moment, I believe this is the latest / greatest and closest to what will be the offical release.

Reply:
fyi @cffls  this has been updated